src/core_plugins/kibana/public/management/styles/main.less

```diff
@@ -72,14 +74,34 @@ kbn-management-landing {
   }
 
   .management-section-info-expanded {
-    .panel-heading-icon {
-      color: @black;
-    }
-
     .management-section-info {
       display: block;
     }
   }
+
+  .management-panel__heading-icon {
+    width: 32px;
+    height: 32px;
+    background-position: 3px 3px;
+    background-repeat: no-repeat;
+    background-size: 26px 26px;
+  }
```

Comment:
Can we tweak these styles?

```less
/**
 * 1. Set icon size.
 * 2. Hack the panel title to move to the left a bit.
 */
.management-panel__heading-icon {
  width: 20px; /* 1 */
  height: 20px; /* 1 */
  background-position: center;
  background-repeat: no-repeat;
  background-size: contain; /* 1 */
  margin-right: -3px; /* 2 */
}
```

This makes the icons slightly smaller and reduces the gap between the icon and the text in the panel header:

![image](https://cloud.githubusercontent.com/assets/1238659/23520812/53e026a0-ff31-11e6-988b-944ea25d99af.png)